deploy/olm-catalog/service-telemetry-operator/manifests/service-telemetry-operator.clusterserviceversion.yaml

```diff
@@ -169,7 +169,6 @@ metadata:
     description: Service Telemetry Framework. Umbrella Operator for instantiating
       the required dependencies and configuration of various components to build a
       Service Telemetry platform for telco grade monitoring.
-    olm.properties: '[{"type": "olm.maxOpenShiftVersion", "value": "4.12"}]'
```

Comment:
This gets moved out of the CSV directly now in favour of it being loaded from the `properties.yaml`. If you run the `operator-sdk-1.26.0 bundle validation` it will WARN you about having these contents inline within the CSV itself.